content/02.main-text.md

```diff
@@ -69,3 +69,31 @@ The randomization procedure was shared with the authors in advance (pre-register
 Given the same author contributions, it always produced the same ordered author list.
 We annotated the author list to indicate that author order was partly randomized and emphasize that the order did not indicate one author contributed more than another from the same category.
 `TODO: In Discussion, present alternative author ordering strategies and literature on contribution in collaborative projects`
+
+### Discussion
+
+Many others have embraced open science principles and piloted open approaches toward drug discovery [@doi:10.1101/087619; @doi:10.1371/journal.pmed.1002276], data management [@doi:10.1371/journal.pbio.1001195; @doi:10.1038/sdata.2016.18; @doi:10.1038/nbt.3516], and manuscript review [@doi:10.12688/f1000research.12037.1].
+`TODO: need help deciding what related topics to include here and which references to use, these are arbitrary examples`
+`TODO: more ideas in doi:10.7287/peerj.preprints.2711v2`
+Several of these open science efforts are GitHub-based like our collaborative writing process.
+The Journal of Open Source Software [@arxiv:1707.02264] and ReScience [@arxiv:1707.04393] journals rely on GitHub for peer review and hosting.
+`TODO: describe Manubot related work here?` [@doi:10.7717/peerj-cs.112; @url:https://github.com/ewanmellor/gh-publisher]
+
+There are potential limitations of our GitHub-based approach.
+Because our review manuscript pertained to a computational topic, most of the authors had computational backgrounds, including previous experience with version control workflows and GitHub.
+In other disciplines, collaborative writing via GitHub and Manubot could present a steeper barrier to entry and deter participants.
+In addition, git carefully tracks all revisions to the manuscript text but not the surrounding conversations that take place through GitHub issues and pull requests.
+These discussions must be archived to ensure that important decisions about the manuscript are preserved and authors receive credit for intellectual contributions that are not directly reflected in the manuscript's text.
+GitHub supports programmatic access to issues, pull requests, and reviews so tracking these conversations is feasible in the future.
+`TODO: did not discuss conflict management and diversity, which are in the outline`
```

Comment:
Does conflict management mean technical challenges with merges, or do you mean disagreements among authors?

I think we need to discuss diversity here, though perhaps in its own paragraph. We should emphasize the importance of an `open science` culture that is inclusive and welcoming if we want to maximize the value of open publishing platforms. If we fail to develop that culture, then open authoring efforts and platforms will miss out on many voices.

Reply:
I think this is more than a limitation of the GitHub based approach, so I'd advocate a new paragraph. Do you want to draft it, or leave a stub and I can try?

Reply:
I meant disagreement among authors and other social aspects.

I added a stub for you.  What you wrote above is what I was thinking but better articulated.